contributors/design-proposals/cpu-manager.md

```diff
@@ -182,6 +188,10 @@ node. Once allocated at pod admission time, an exclusive CPU remains
 assigned to a single container for the lifetime of the pod (until it
 becomes terminal.)
 
+The Kubelet requires `kubereserved.cpu` plus `systemreserved.cpu` to be
```

Comment:
s/requires `kubereserved.cpu` plus `systemreserved.cpu`/requires the sum of `kubereserved.cpu` and `systemreserved.cpu`